dockerfiles/rbe-ubuntu22-04/Dockerfile

```diff
@@ -0,0 +1,121 @@
+FROM marketplace.gcr.io/google/ubuntu2204@sha256:b70be2d80535d3fe99e2bc484133da588728add8fc15956d124c478b42e61f2c
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Python 2 and 3
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+      python2 \
+      python2-dev \
+      python3 \
+      python3-dev \
+      python-is-python3 \
+      && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# JRE / JDK (headless).
+# Don't install recommended packages since that would include
+# some alsa/a11y packages which aren't needed for server installs.
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+      default-jre-headless \
+      default-jdk-headless \
+      && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
```

Comment:
I think we could remove Java from newer images - Bazel's remote JDK works well and is easy to use

```suggestion
```